### DISCUSSION TITLE
[open-graph-scraper] Make OpenGraphImage fields optional

### DISCUSSION THREAD

**Discussion by Dichotomia** [DefinitelyTyped/DefinitelyTyped#62674] [Issues with a @types package]
Hello.

The library seems to enable height and width to be null on OpenGraphImage object.

Please reflect this on typings.

Best regards

[a comment by an automated account is not shown]

**Comment by Semigradsky** [comment]
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/62739
